resources/assets/components/HistoryModal/index.js

```diff
@@ -76,4 +77,12 @@ class HistoryModal extends React.Component {
   }
 }
 
+HistoryModal.propTypes = {
```

Comment:
I have no idea why this is red. If anyone has an idea, let me know!

Reply:
Ahhh, a classic! Github's JSX syntax highlighting [is broken](https://git.io/vF8fk).

Reply:
oh fun!